config/configLoadHelpers.go

```diff
@@ -240,3 +241,21 @@ func readConfigInto(dest any, location string, opts *CmdEnv) (string, error) {
 
 	return hash, nil
 }
+
+// ConfigHashMetrics takes a config hash and returns a decimal value for use in metrics.
+// The decimal value is the last 4 characters of the config hash, converted to decimal.
+// If the config hash is too short, or if there is an error converting the hash to decimal,
+// an error is returned.
+func ConfigHashMetrics(hash string) (int64, error) {
```

Comment:
You've got a lot of code here to return errors in situations that don't really matter, and I'd prefer not to add it. If we can't load the config, we'll tell people that where it was loaded. If we can, it's guaranteed to have a valid hexadecimal hash.  I'd like to see this routine only return an int value and no error -- 0 if the hash is too short (probably empty) and the decimal value of the last 4 digits of the hash if it's not. I would just ignore the error return from parseInt. 

Then you don't have to handle errors when saving it as a metric and the code in app.go gets shorter too, and the test code for this function gets simpler.